vlib/v/checker/tests/inout/match_expr_else.vv

```diff
@@ -10,4 +10,5 @@ fn main() {
 			'string'
 		}
 	}
+	_ = res
```

Comment:
You can use `_` directly instead of `res` in `match` statement, or even skip it (this is better, if it won't break tests).

Reply:
Are you sure that won't result in the function being removed in the generated C code? @alexesprit

Reply:
Why should this function be removed?

Reply:
I believe gcc removes dead code when it has certain flags turned on

Reply:
This code isn't supposed to be compiled by V, so `gcc` won't remove dead code.

Plus, it's still dead code, as the `res` variable is not used, and that means the whole match statement is useless. Compilers are smart to find such cases.

Reply:
Alright, I'll remove these instances.